src/renderers/environments.js

```diff
@@ -30,8 +30,14 @@ class EnvironmentsRenderer {
         break;
       case 'table':
         environments.forEach((environment) => {
-          table.push([environment.id, environment.name, environment.apiEndpoint,
-            environment.type]);
+          table.push([
+            environment.id,
+            environment.name,
+            // when the onboarding is not finished, the apiEndpoint is not available.
```

Comment:
```suggestion
```

I'm not in that specific case but I still have the issue. I don't think this comment is relevant here.